addons/web/static/tests/views/kanban_view_tests.js

```diff
@@ -10965,4 +10966,53 @@ QUnit.module("Views", (hooks) => {
             productFieldName + "..."
         );
     });
+
+    QUnit.test("drag record on folded column", async (assert) => {
+        let searchReadProm;
```

Comment:
is this the same thing as testing the problem with a column that was never loaded?

Reply:
looking at model.transaction --> onCommit seems to be unused --> check if things cannot be simplified in another commit